packages/components/text/src/Text.tsx

```diff
@@ -69,7 +69,7 @@ export const Text = compressible<TextProps, TextTokens>((props: TextProps, useTo
     color,
     variant,
     fontFamily: font == 'base' ? 'primary' : font,
-    fontSize: globalTokens.font.size[size],
+    fontSize: globalTokens.font['size' + size],
     fontWeight: globalTokens.font.weight[weight] as FontWeightValue,
```

Comment:
Considering this is one of our active V1 controls we're about to ship, could we update this so it doesn't use the non-type-safe method of using `[]` to access properties? @rurikoaraki @lenahong fyi

Reply:
To be clear, I don't think this is blocking this change.

Reply:
Side note that may or may not be helpful at all: TypeScript recently added a weird little thing where you can [dynamically build types using a `${string}_interpolation_${pattern}`](https://www.typescriptlang.org/docs/handbook/2/template-literal-types.html).

Reply:
Made a task for this on ADO, going to resolve this conversation. If anyone has any thoughts on using 'size' + size in this PR feel free to reopen